kaffe/tensorflow/network.py

```diff
@@ -177,7 +177,7 @@ def lrn(self, input, radius, alpha, beta, name, bias=1.0):
 
     @layer
     def concat(self, inputs, axis, name):
-        return tf.concat(concat_dim=axis, values=inputs, name=name)
+        return tf.concat(axis=axis, values=inputs, name=name)
```

Comment:
@PalashTanejaPro We don't really need to make any change other than this line because we are not using the `examples` and we shouldn't try to rename variables because it will require extensive testing to check if it doesn't break anything else. So just make this change and let the others be as they were.

Reply:
Sure thing.